lib/net/pool.js

```diff
@@ -1717,8 +1671,6 @@ class Pool extends EventEmitter {
         }
       }
     }
-
-    this.fillOutbound();
```

Comment:
we're totally sure this is OK? We fill outbound enough without triggering here?

Reply:
Yeah, we have `refillTimer` always running with `Pool.REFILL_INTERVAL` which is set to 3 seconds.